### PR TITLE
Add AltID Login to HRD policy

### PR DESCRIPTION
**Pull request by hpsin**

### CODE REVIEW

**Comment by hpsin** [comment]
@calvinlui as FYI.

**Comment by hpsin** [comment]
@davidmu1 The example API calls for this are all pretty sparse (e.g. don't even have the right policy type name) - is that something that can be automatically updated or does it need to be manually updated in every spot?

**Comment by hpsin** [comment]
Why does type "JSON" trip up the api doctor with:
  Error: microsoft.graph.homeRealmDiscoveryPolicy/AlternateIdLogin:
      Referenced type microsoft.graph.jSON is not defined in the doc set! Potential suggestion: microsoft.graph.Json

JSON != jSON != Json , so not sure where jSON came from.

**Review by dkershaw10 — APPROVED** [review]
LGTM
You'll need to create a manual changelog entry here.  This kind of change cannot be automatically generated, because the API schema has not changed :(

**Comment by hpsin** [comment]
Thanks Dan! Eric helped me out with the change log, I'll get that in today.

**Comment by dkershaw10** [comment]
@hpsin Adding more examples needs to be done manually - especially in this case as half the API here is not schematized.

**Review by davidmu1 — APPROVED** [review]

File: changelog/Manual.NonWorkloadChanges.json [hunk 1]
```diff
@@ -1,5 +1,41 @@
 {
   "changelog": [
+    {
+          "ChangeList": [
+        {
+          "Id": "3516a214-222c-4462-9a86-8ca089325e99",
+          "ApiChange": "Property",
+          "ChangedApiName": "definition",
+          "ChangeType": "Addition",
+          "Description": "Added the **AlternateIdLogin** field to the [homeRealmDiscoveryPolicy](/graph/api/resources/homerealmdiscoverypolicy?view=graph-rest-beta) policy definition.",
```
Comment (dkershaw10) [inline changelog/Manual.NonWorkloadChanges.json:10]: policy **definition** property.

Comment (dkershaw10) [inline changelog/Manual.NonWorkloadChanges.json:10]: Please look at the other examples.  In the JSON file you need a **fully** qualified path (and with en-us)
So prepend what you have with: "https://docs.microsoft.com/en-us/"

File: changelog/Manual.NonWorkloadChanges.json [hunk 2]
```diff
@@ -1,5 +1,41 @@
...
+          "Target": "homerealmdiscovery"
+        }
+      ],
+      "Id": "3516a214-222c-4462-9a86-8ca089325e99",
+      "Cloud": "prd",
+      "Version": "beta",
+      "CreatedDateTime": "2020-10-02T00:00:00",
+      "WorkloadArea": "Identity and access",
```
Comment (dkershaw10) [inline changelog/Manual.NonWorkloadChanges.json:18]: Please add these entries to the Microsoft.DirectoryServices.json file.

Comment (hpsin) [reply changelog/Manual.NonWorkloadChanges.json:18]: Thanks! How should I have figured out to put it there, for next time? DirectoryServices doesn't appear in the spreadsheet I was sent

**Comment by hpsin** [comment]
#sign-off ?

**Comment by hpsin** [comment]
#please-merge ?

**Comment by eketo-msft** [comment]
> #please-merge ?

https://msazure.visualstudio.com/One/_wiki/wikis/Microsoft%20Graph%20Partners/55090/Update-existing-documentation?anchor=request-a-content-review

Request a content review
When the content has been reviewed and revised to be technically accurate, set the label to content review. The content review can include a content developer, but always includes the person who merges PRs. The PR is reviewed for consistency, compliance, globalization, adherence to guidelines, etc.

After the review process is complete, incorporate any feedback that was provided in the PR. When all feedback has been incorporated, reviewers have signed off, and the APIs have been released live, **_set the label to ready to merge_**, and then the topics are published.

**Comment by hpsin** [comment]
Thanks @eketo-msft .  I've already had this approved by two people, who would presumably also be the people I ask to review it again? I've also just now asked for access to the repo since I have no rights to change tags or add reviewers here.

**Comment by eketo-msft** [comment]
> Thanks @eketo-msft . I've already had this approved by two people, who would presumably also be the people I ask to review it again? I've also just now asked for access to the repo since I have no rights to change tags or add reviewers here.

Ah, I didn't realize there were access issues.  Assuming you incorporated their feedback then its fine to move forward.  They shouldn't need to review again if they signed off - or they should have waited for updates rather than approved with the suggested changes. 

I'll mark it ready for review.

**Review by eketo-msft — APPROVED** [review]

**Comment by hpsin** [comment]
Thanks! They're unblocked now, since I got approved for the group membership.

File: api-reference/beta/resources/homerealmdiscoverypolicy.md [hunk 3]
```diff
@@ -13,7 +13,7 @@ Namespace: microsoft.graph
...
+Represents a policy to control Azure Active Directory authentication behavior for federated users, in particular for auto-acceleration and user authentication restrictions in federated domains. You can set homeRealmDiscoveryPolicy for all service principals in your organization, or for specific service principals in your organization.  For more scenario and policy details see [Configure Azure AD sign in behavior for an application by using a Home Realm Discovery policy](https://docs.microsoft.com/azure/active-directory/manage-apps/configure-authentication-for-federated-users-portal) as well as [Sign-in to Azure Active Directory using email as an alternate login ID](https://docs.microsoft.com/azure/active-directory/authentication/howto-authentication-use-email-signin).
```
Comment (Lauragra) [inline api-reference/beta/resources/homerealmdiscoverypolicy.md:17]: Global: For all links, please make them relative, so, delete the "https://docs.microsoft.com" part.

**Comment by hpsin** [comment]
@Lauragra  thank you! 
Fixed #microsoftgraph/microsoft-graph-docs/9135

**Pull request merged**